crucible-mir/src/Mir/Generator.hs

```diff
@@ -657,6 +657,20 @@ subfieldRef ::
   MirGenerator h s ret (R.Expr MIR s MirReferenceType)
 subfieldRef ctx ref idx = G.extensionStmt (MirSubfieldRef ctx ref idx)
 
+-- | Extend the reference path encapsulated in the provided `Expr MIR s
+-- MirReferenceType` (i.e. `ref`), which should terminate in a struct, to
```

Comment:
Haddock nit pick: surrounding text with backticks (or single quotes) in Haddock (`` `...` ``) is meant for hyperlinking an individual identifier. I think you want to use `@...@` here instead, which simply renders the text in monospace font:

```suggestion
-- | Extend the reference path encapsulated in the provided @Expr MIR s
-- MirReferenceType@ (i.e. @ref@), which should terminate in a struct, to
```

And similarly for other uses of backticks in this PR.

Reply:
I did, thanks!